homeassistant/components/demo/__init__.py

```diff
@@ -244,10 +244,17 @@ def _generate_mean_statistics(start, end, init_value, max_diff):
     return statistics
 
 
-def _generate_sum_statistics(start, end, init_value, max_diff):
+async def _insert_sum_statistics(hass, metadata, start, end, max_diff):
```

Comment:
Since I'm trying to add demo to strict typing (see #77186), could you add the type hint here?
```python
async def _insert_sum_statistics(hass: HomeAssistant, metadata: StatisticsMetaData, start: datetime.datetime, end: datetime.datetime, max_diff: float) -> None:
```

Reply:
Let's do that in #77186 instead 👍

Reply:
OK, I'll rebase after this is merged.